src/sentry/api/endpoints/group_details.py

```diff
@@ -260,6 +256,29 @@ def get(self, request: Request, group) -> Response:
                 }
             )
 
+            participants = user_service.serialize_many(
+                filter={"user_ids": GroupSubscriptionManager.get_participating_user_ids(group)},
+                as_user=request.user,
+            )
+
+            for participant in participants:
+                participant["type"] = "user"
+
+            if features.has("organizations:team-workflow-notifications", group.organization):
+                team_ids = GroupSubscriptionManager.get_participating_team_ids(group)
+
+                teams = Team.objects.filter(id__in=team_ids)
+
+                serialized_teams = []
+                for team in teams:
+                    serialized_team = serialize(team, request.user, TeamSerializer())
```

Comment:
we can take `TeamSerializer()` out and just declare it under teams as a variable and re-use it instead of re-initializing it every loop